tests/storage/test_main.py

```diff
@@ -0,0 +1,46 @@
+# -*- coding: utf-8 -*-
```

Comment:
Files containing admin API unit tests should live at `tests/rest/admin/` folder. There's already a `test_user.py` file in there actually.

Could you add your test to the `UsersListTestCase` class instead instead of this new file? Feel free to add your copyright to that file as you've modified it.

Reply:
I did indeed update the `UsersListTestCase`, but this test is not for the REST API, but for the storage API.